xmtp_mls/src/storage/encrypted_store/sqlcipher_connection.rs

```diff
@@ -228,14 +228,15 @@ impl super::native::ValidatedConnection for EncryptedConnection {
             cipher_version.first().as_ref().map(|v| &v.cipher_version),
             cipher_provider_version
         );
-
-        if tracing::enabled!(tracing::Level::INFO) {
+/*
+        if tracing::enabled!(tracing::Level::DEBUG) {
             conn.batch_execute("PRAGMA cipher_log = stderr; PRAGMA cipher_log_level = INFO;")
                 .ok();
         } else {
+*/
             conn.batch_execute("PRAGMA cipher_log = stderr; PRAGMA cipher_log_level = WARN;")
                 .ok();
-        }
+        // }
```

Comment:
Are these changes deliberate?

Reply:
Yeah the SQLCipher logs are a little annoying when generating lots of groups, it just spits out that the logs were enabled so I commented it out temporarily